spec/person_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../person'
+
+describe Person do
+  context 'When creating a person' do
+    person = Person.new(age: 30, parent_permission: false, name: 'Susan')
+    it 'Creates a new person with name Susan' do
+      expect(person).to be_an_instance_of Person
+    end
+
+    it 'Age of new pwerson to equal 30' do
+      expect(person.age).to eq 30
+    end
+  end
+end
```

Comment:
- [ ] please be more thorough when testing the classes, for example, if the name will be capitalized and shortened to 10 characters in case it is not and what if the parent permission is true and when it is false.

Reply:
Hello @mikethreels  could you please shade more light on what you require me to do on the changes you have asked us to make because as far as I'm concerned all tests are running perfectly why. So what is wrong in how we implemented the test?

Reply:
Shade more light because in this instance a person can be a student or a teacher so I dont really understand where the problem is ??

Reply:
Hi,
Your test are setup correctly in general
The Person however always has parent_permission which can be true or false, therefore you need to test it. Also corrector method needs to be tested through the person class to check if everything is setup correctly.
Hope this will make it clearer for you

Reply:
That was implemented in the student file

Reply:
But anyway let me try and do what you asking us to do

Reply:
That's great, It is always best practice to be thorough when testing as a small mistake can make a difference you are not expecting. I'm confident looking at the tests you wrote so far you will have no issues including these changes. 🥇

Reply:
@mikethreels thank for the review